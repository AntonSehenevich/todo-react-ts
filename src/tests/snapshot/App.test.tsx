import { Provider } from 'react-redux'
import renderer from 'react-test-renderer'
import App from '../../components/app'
import setupStore from '../../store'

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={setupStore()}>
        <App />
      </Provider>
    )
    .toJSON()

  expect(tree).toMatchSnapshot()
})
