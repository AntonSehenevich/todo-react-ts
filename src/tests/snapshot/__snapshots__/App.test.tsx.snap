// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="layout"
>
  <div
    className="content"
  >
    <div
      className="container"
    >
      <svg
        className="logo"
        data-testid="logo-icon"
        height={70}
        viewBox="-10 10 50 40"
        width={50}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M 5 45 H 25 V 5"
          fill="transparent"
          stroke="currentColor"
          strokeLinecap="round"
          strokeWidth="2"
          transform="rotate(45 15 25)"
        />
      </svg>
      <h3
        className="title"
      >
        TODO
      </h3>
    </div>
    <div
      className="container"
    >
      <div
        className="container"
        data-testid="add-todo"
      >
        <input
          className="input"
          onBlur={[Function]}
          onChange={[Function]}
          onKeyDown={[Function]}
          placeholder="Something to do?"
          type="text"
          value=""
        />
      </div>
    </div>
  </div>
</div>
`;
